src/components/EditableField/EditableField.Input.tsx

```diff
@@ -326,7 +326,7 @@ export class EditableFieldInput extends React.Component<InputProps> {
             innerRef={this.setInputNode}
             name={name}
             placeholder={placeholder}
-            type={type === 'email' ? 'text' : type}
+            type={type === 'password' ? type : 'text'}
```

Comment:
we still need to make sure that when it is email we change it to text. I think it was to fix an obscur bug in chrome

something like that could work:
```suggestion
            type={['password', 'email'].includes(type) ? type : 'text'}
```

Reply:
@plbabin do you mean when it is email we should leave it as email? Currently, it changes everything except password to text.